examples/chip-tool/commands/delay/BusyWaitCommand.h

```diff
@@ -0,0 +1,55 @@
+/*
+ *   Copyright (c) 2022 Project CHIP Authors
+ *   All rights reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ *
+ */
+
+#pragma once
+
+#include "../common/CHIPCommand.h"
+
+/**
+ * This command block the event loop processing for a given amount of time.
+ *
+ * For example when the event loop is blocked the messages coming-in will not be acked,
+ * forcing a retransmission on the other side.
+ *
+ */
+
+class BusyWaitCommand : public CHIPCommand
+{
+public:
+    BusyWaitCommand(CredentialIssuerCommands * credIssuerCommands) : CHIPCommand("busy-wait", credIssuerCommands)
```

Comment:
And here, replace busy-wait with something else?

Maybe just call this SleepCommand and use "sleep" for the actual command invocation?

Reply:
I changed it for `SleepCommand`.